2_clean_sample.yml

```diff
@@ -18,12 +18,11 @@ sources:
 targets:
   2_clean_sample:
     depends: 
-      - clean_sample_data
       - sample_data
   
-  clean_sample_data: 
+  clean_sample: 
```

Comment:
r objects are interesting in the context of a shared cache, because anything that has `clean_sample` as a parent or ancestor will need to build `clean_sample` locally to know whether the downstream item is up to date. this is true even if the downstream item is an indicator file representing the status of a google drive folder. so if `clean_sample` is slow to build, you might want to make it a file with indicator file rather than an R object. but if it's not too costly for everybody to build this object locally, then go for it. just wanted to note that there's a tradeoff here that may matter sometimes.

Reply:
oh......I didn't realize it was bi-directional. So just to clarify... if I use `clean_sample` later in a target that then produces an indicator file (and therefore pushes it up to google drive), then the other person would have still needed to download and create that intermediate R object `clean_sample`...even though they can get the final product from google drive. Is that right? So then yeah, seems like every intermediate thing needs to be pushed up to google drive (correct?) Hmmmmm....OK....